### ISSUE TITLE
Status about using GDC as host D compiler / bootstrapping

### ISSUE THREAD

**Issue by diizzyy** [ldc-developers/ldc#4232]
I found conflicting information about this matter so I figured I might as well ask.
 
Wiki states (with the disclaimer that it can be outdated) that GDC is fine (https://wiki.dlang.org/Building_LDC_from_source)
Support got ripped out a few years ago as far as I can tell, https://github.com/ldc-developers/ldc/commit/3f7cf7520033a225d4c12390b4a1ff68f3d8257c but there are multiple issues mentioning GDC after this commit which makes me a bit confused about this matter. If there are any patches etc feel free to point me in that direction.

Tagging @dbankov-vmware @lenoil98 as you seem to be trying to use GDC.

What I'm trying to determine is if it's possible to use GCC 11 for bootstrapping in some way.

Best regards,
Daniel

**Comment by thewilsonator** [comment]
> Support got ripped out a few years ago as far as I can tell, https://github.com/ldc-developers/ldc/commit/3f7cf7520033a225d4c12390b4a1ff68f3d8257c

that removed the diagnostics for  building with `gdc` and `ldc2` compilers specifically, not their wrappers `gdmd` and `ldmd`, which translate the `dmd` style arguments and call their corresponding main compilers with the appropriate arguments.

**Comment by kinke** [comment]
Yep, GDC as host compiler works fine - there are CI jobs using GDC 10 and 11 as host compilers, via the mentioned [`gdmd`](https://github.com/D-Programming-GDC/gdmd) wrapper (Ubuntu package `gdmd`).
